README.md

```diff
@@ -16,16 +16,16 @@
 
 The Pi-hole® is a [DNS sinkhole](https://en.wikipedia.org/wiki/DNS_Sinkhole) that protects your devices from unwanted content without installing any client-side software.
 
-- **Easy-to-install**: our dialogs walk you through the simple installation process in less than ten minutes
+- **Easy-to-install**: Our dialogs walk you through the simple installation process in less than ten minutes
 - **Resolute**: content is blocked in _non-browser locations_, such as ad-laden mobile apps and smart TVs
 - **Responsive**: seamlessly speeds up the feel of everyday browsing by caching DNS queries
 - **Lightweight**: runs smoothly with [minimal hardware and software requirements](https://docs.pi-hole.net/main/prerequisites/)
 - **Robust**: a command-line interface that is quality assured for interoperability
-- **Insightful**: a beautiful responsive Web Interface dashboard to view and control your Pi-hole
-- **Versatile**: can optionally function as a [DHCP server](https://discourse.pi-hole.net/t/how-do-i-use-pi-holes-built-in-dhcp-server-and-why-would-i-want-to/3026), ensuring _all_ your devices are protected automatically
+- **Insightful**: a beautiful responsive Web interface dashboard to view and control your Pi-hole
+- **Versatile**: it can optionally function as a [DHCP server](https://discourse.pi-hole.net/t/how-do-i-use-pi-holes-built-in-dhcp-server-and-why-would-i-want-to/3026), ensuring _all_ your devices are protected automatically
```

Comment:
Why start `Our` with a capital letter above, but but no capital letter for `it` here? 

Likewise, if one is starting `Our` with a capital letter, why not `Content`/`Seamlessly`/`Runs`/`A`/`Capable`/`Blocks`/`Open-source` etc

Reply:
Some words (like proper nouns/adjectives) should be capitalized, while others (like common nouns/adjectives) should not.

Reply:
https://homework.study.com/explanation/is-our-a-proper-noun.html 🤷‍♂️